src/main/java/org/isf/patconsensus/rest/PatientConsensusController.java

```diff
@@ -0,0 +1,95 @@
+
+import java.util.Optional;
+
+import org.isf.patconsensus.dto.PatientConsensusDTO;
+import org.isf.patconsensus.manager.PatientConsensusBrowserManager;
+import org.isf.patconsensus.mapper.PatientConsensusMapper;
+import org.isf.patconsensus.model.PatientConsensus;
+import org.isf.shared.exceptions.OHAPIException;
+import org.isf.utils.exception.OHServiceException;
+import org.isf.utils.exception.model.OHExceptionMessage;
+import org.isf.utils.exception.model.OHSeverityLevel;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.Authorization;
+
+@RestController()
+@RequestMapping("/patientConsensus")
+@Api(value = "/patientConsensus", produces = MediaType.APPLICATION_JSON_VALUE, authorizations = { @Authorization(value = "apiKey") })
+public class PatientConsensusController {
+
+	private static final Logger LOGGER = org.slf4j.LoggerFactory.getLogger(PatientConsensusController.class);
+
+	@Autowired
+	protected PatientConsensusBrowserManager manager;
+
+	@Autowired
+	protected PatientConsensusMapper mapper;
+
+	@GetMapping(value = "/{patientId}", produces = MediaType.APPLICATION_JSON_VALUE)
+	ResponseEntity<PatientConsensusDTO> getPatientConsensus(@PathVariable Integer patientId) throws OHServiceException {
+		LOGGER.info("Retrieving patient consensus: {}", patientId);
+		Optional<PatientConsensus> patientConsensus = manager.getPatientConsensusByUserId(patientId);
+		if (patientConsensus.isEmpty()) {
+			return ResponseEntity
+							.status(HttpStatus.NO_CONTENT).body(null);
+		}
+		PatientConsensusDTO patientDTO = mapper.map2DTO(patientConsensus.get());
+		return ResponseEntity.ok(patientDTO);
+	}
+
+	@PutMapping(value = "/{patientId}", produces = MediaType.APPLICATION_JSON_VALUE)
+	ResponseEntity<PatientConsensusDTO> updatePatientConsensus(@PathVariable Integer patientId, @RequestBody PatientConsensusDTO patientConsensus)
+					throws OHServiceException {
+		LOGGER.info("Update patient consensus by id: {}", patientId);
+		if (patientId != patientConsensus.getPatientId()) {
+			throw new OHAPIException(new OHExceptionMessage(null, "Patient code mismatch", OHSeverityLevel.ERROR));
+		}
+		Optional<PatientConsensus> patConsensusOpt = this.manager.getPatientConsensusByUserId(patientId);
+		if (patConsensusOpt.isEmpty()) {
+			throw new OHAPIException(new OHExceptionMessage(null, "PatientConsensus not found!", OHSeverityLevel.ERROR));
```

Comment:
```suggestion
			throw new OHAPIException(new OHExceptionMessage("PatientConsensus not found."));
```
New constructor and don't finish messages with `!`